src/connectors/github_organization.py

```diff
@@ -0,0 +1,251 @@
+                    , value:repository::VARIANT repository
+                    , value:pusher::VARIANT pusher
+                    , value:organization::VARIANT organization
+                    , value:sender::VARIANT sender
+                    , value:action::VARCHAR(256) action
+                    , value:check_run::VARIANT check_run
+                    , value:check_suite::VARIANT check_suite
+                    , value:number::NUMBER(38,0) number
+                    , value:pull_request::VARIANT pull_request
+                    , value:label::VARIANT label
+                    , value:requested_team::VARIANT requested_team
+                    , value:ref_type::VARCHAR(256) ref_type
+                    , value:master_branch::VARCHAR(256) master_branch
+                    , value:description::VARCHAR(256) description
+                    , value:pusher_type::VARCHAR(256) pusher_type
+                    , value:review::VARIANT review
+                    , value:changes::VARIANT changes
+                    , value:comment::VARIANT comment
+                    , value:issue::VARIANT issue
+                    , value:id::NUMBER(38,0) id
+                    , value:sha::VARCHAR(256) sha
+                    , value:name::VARCHAR(256) name
+                    , value:target_url::VARCHAR(8192) target_url
+                    , value:context:: VARCHAR(256) context
+                    , value:state:: VARCHAR(256) state
+                    , value:commit::VARIANT commit
+                    , value:branches:VARIANT branches
+                    , value:created_at::TIMESTAMP_LTZ(9) created_at
+                    , value:updated_at::TIMESTAMP_LTZ(9) updated_at
+                    , value:assignee::VARIANT assignee
+                    , value:release::VARIANT release
+                    , value:membership::VARIANT membership
+                    , value:alert::VARIANT alert
+                    , value:scope::VARCHAR(256) scope
+                    , value:member:VARIANT member
+                    , value:requested_reviewer::VARIANT requested_reviewer
+                    , value:team::VARIANT team
+                    , value:starred_at::TIMESTAMP_LTZ(9) starred_at
+                    , value:pages::VARIANT pages
+                    , value:project_card::VARIANT project_card
+                    , value:build::VARIANT build
+                    , value:deployment_status::VARIANT deployment_status
+                    , value:deployment::VARIANT deployment
+                    , value:forkee::VARIANT forkee
+                    , value:milestone::VARIANT milestone
+                    , value:key::VARIANT key
+                    , value:project_column::VARIANT project_column
+                    , value:status::VARCHAR(256) status
+                    , value:avatar_url::VARCHAR(256) avatar_url
+                FROM data.{base_name}_stream
+                '''
+        )
+    )
+
+    db.insert(
```

Comment:
This db.insert() call seems to be floating; it'll only get called once as part of the connection finalization process, which is likely not what we want to do.

Reply:
Ya I wasn't quite sure how to use the insert function. What would I put in there? Because I'm trying to go off of how the Okta connector inserts, but there are some values that aren't related to this connector.

Reply:
What other values are you trying to insert? The Okta connector uses db.insert because it scrapes data from an API, then processes that data itself and loads it into snowflake; data is never staged or stored anywhere. Here, we have data stored in S3, so we don't need to just raw insert it ourselves; the pipe definition will serve the function of parsing data, doing minor transformations, and loading it into the table.